<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoStream</class>
 <widget class="QMainWindow" name="VideoStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="videoLable">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="streamStopBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消监控</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorOpenBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>远程开门</string>
    </property>
   </widget>
   <widget class="QLabel" name="accessImageL">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="faceLibImageL">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="streamReqBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始监控</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
